src/test/java/edu/cornell/kfs/tax/fixture/TaxOutputDefinition.java

```diff
@@ -0,0 +1,31 @@
+package edu.cornell.kfs.tax.fixture;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import edu.cornell.kfs.sys.util.FixtureUtils;
+import edu.cornell.kfs.tax.batch.xml.TaxOutputDefinitionV2;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.FIELD)
+public @interface TaxOutputDefinition {
```

Comment:
Do you think it would be appropriate to rename this class/file to TaxOutputDefinitionFixture?

Reply:
I was originally trying to avoid clutter in the unit tests, but I agree that a "Fixture" suffix would help clarify that the annotation represents a fixture. I'll make adjustments accordingly.